packages/fxa-settings/src/lib/auth-errors/auth-errors.ts

```diff
@@ -604,13 +620,41 @@ const ERRORS = {
 };
 
 type ErrorKey = keyof typeof ERRORS;
-type ErrorVal = { errno: number; message: string };
+type ErrorVal = { errno: number; message: string; version?: number };
+
+export const composeAuthUiErrorTranslationID = (err: {
+  errno?: number;
+  message: string;
+}) => {
+  /* all of these checks for fields/values being present look a little wonky, but allow us to work with
+   * the AuthUiError type, which has all optional fields. Previously this wasn't an issue bc we didn't use
+   * a utility.
+   */
+  if (err.errno && err.errno in AuthUiErrorNos) {
+    const error = AuthUiErrorNos[err.errno];
```

Comment:
Since you're passing in the entire error object, we probably don't need this extra lookup on L634. This might be a little slicker:

```
export const composeAuthUiErrorTranslationID = (error: AuthUiError) => {
  /* all of these checks for fields/values being present look a little wonky, but allow us to work with
   * the AuthUiError type, which has all optional fields. Previously this wasn't an issue bc we didn't use
   * a utility.
   */
  if (error.errno && error.errno in AuthUiErrorNos) {
    return `auth-error-${error.errno}${error.version ? '-' + error.version : ''}`;
  }
  /*
   * We opt to return an empty string instead of throwing an error so that this can't break in prod.
   * Instead, we log to sentry.
   */
  const logMessage = error.errno
    ? `composeAuthUiErrorTranslationID: There is no matching error in AuthUiErrors. error: ${JSON.stringify(
        error
      )}`
    : `composeAuthUiErrorTranslationID: No error number given, unable to create a localization ID for AuthUiError string. error: ${JSON.stringify(
        error
      )}`;
  Sentry.captureMessage(logMessage);
  return '';
};
```

Reply:
Just looked into this -- so the lookup is necessary because the "version" object doesn't come in on the error coming from the server. It's only available in the dictionary of errors that we keep client-side, so I'm going to leave it as-is. That being said! I think that's something we can/should fix when we clean up the way the auth errors are handled overall!

Reply:
Oh I see! Do you mind filing a follow up issue for that (you could also go ahead and make a new epic to hold stuff like that that comes up if you want), or alternatively make sure you've got a note to yourself to file it as part as the conversion to React epic? 😎